jwst/tests/base_classes.py

```diff
@@ -163,3 +213,48 @@ def raw_from_asn(asn_file):
             members.append(member['expname'])
 
     return members
+
+
+def _data_glob_local(path, glob='*'):
+    """Perform a glob on the local path
+
+    Parameters
+    ----------
+    path: File path-like object
+        The path to check.
+
+    glob: str
+        The file name match criterion
+
+    Returns
+    -------
+    file_paths: [str[, ...]]
+        Full file paths that match the glob criterion
+    """
+    full_glob = op.join(path, glob)
+    return _sys_glob(full_glob)
+
+
+def _data_glob_url(url, glob='*'):
+    """
+    Parameters
+    ----------
+    url: str
+        The URL to check
+
+    glob: str
+        The file name match criterion
+
+    Returns
+    -------
+    url_paths: [str[, ...]]
+        Full URLS that match the glob criterion
+    """
+    path = requests.get(url).text
+    soup = BeautifulSoup(path, 'html.parser')
```

Comment:
Would be nice if we can use Artifactory's search API here instead of scraping our Artifactory web server.  I'm investigating and testing.

Reply:
you did it. resolved 👍

Reply:
Commit added to use Artifactory's search API.